spring-integration-scripting/src/main/java/org/springframework/integration/scripting/DefaultScriptVariableGenerator.java

```diff
@@ -44,8 +46,8 @@ public DefaultScriptVariableGenerator(Map<String, Object> variableMap) {
 		this.variableMap = variableMap;
 	}
 
-	public Map<String, Object> generateScriptVariables(Message<?> message) {
-		Map<String, Object> scriptVariables = new HashMap<String, Object>();
+	public Map<String, Object> generateScriptVariables(@Nullable Message<?> message) {
```

Comment:
After fixing `ScriptExecutingMessageSource` many of these `@Nullable` on `message` would go away.